network-api/networkapi/templates/donate/pages/help_page.html

```diff
@@ -0,0 +1,14 @@
+{% extends "../pages/base.html" %}
+
+{% block body_id %}help{% endblock body_id %}
+
+{% block content %}
+    <div class="tw-container tw-py-10">
+        <h1 class="tw-h1-heading"> {{ page.title }} </h1>
+        {% if page.show_notice %}
+            {% include "../fragments/help_page_notice.html" %}
+        {% endif %}
+        <p> Form </p>
+        <p> Body </p>
```

Comment:
We do have a `body` field already. Shouldn't we already render it here instead of this placeholder?

Reply:
Hi @jhonatan-lopes! That work is being taken care of in ticket https://github.com/MozillaFoundation/foundation.mozilla.org/issues/11178. Thanks! 👍